backend/apps/github/models/issue_comment.py

```diff
@@ -0,0 +1,35 @@
+"""GitHub app issue comment model."""
+
+from django.db import models
+
+
+class IssueComment(models.Model):
+    """Represents a comment on a GitHub issue."""
+
+    github_id = models.BigIntegerField(unique=True)
+    issue = models.ForeignKey("github.Issue", on_delete=models.CASCADE, related_name="comments")
+    author = models.ForeignKey(
+        "github.User", on_delete=models.SET_NULL, null=True, related_name="issue_comments"
+    )
+    body = models.TextField()
+    created_at = models.DateTimeField()
+    updated_at = models.DateTimeField()
+
+    @classmethod
+    def update_data(cls, gh_comment, issue_obj, author_obj):
```

Comment:
This is not how `update_data` works. Please check other model examples. This needs bulk save support and should never create an object unless `save=True` is passed.